### DISCUSSION TITLE
Please tell me how to visualize specific grains which have particular grain IDs

### DISCUSSION THREAD

**Discussion by nippyawn** [neperfepx/neper#872] [Q&A] [answered]
Hello,

I want to visualize specific grains which have particular grain IDs.
 I tried the command below, but the correct grains don't visualize:

```
neper -V n50-id1.tess,n50-id1.msh -showcell "file(grain-ids.txt)" -print img
```
The actual number of grains I want to visualize is about 1000.
Please tell me how to visualize specific grains which have particular grain IDs
The format of "grain-ids.txt" is as follows:
```
2
3
5
6
```

**Comment by rquey** [comment, marked as answer]
The file must contain 1000 entries (one for each grain) equal to 0 (to hide a grain) or 1 (to show a grain). See [-showcell](https://neper.info/doc/neper_v.html#cmdoption-show-cell-poly-face-edge-ver-seed-crystal-faceinter-vox).

**Reply by nippyawn** [reply]
Thank you! I was able to solve the problem using that method.
